addons/editor-square-inputs/userscript.js

```diff
@@ -0,0 +1,48 @@
+export default async function ({ addon, console }) {
+  const ScratchBlocks = await addon.tab.traps.getBlockly();
+  const vm = addon.tab.traps.vm;
+
+  const opcodeToSettings = {
+    text: "text",
+    argument_editor_string_number: "text",
+    colour_picker: "color",
+  };
+
+  const originalJsonInit = ScratchBlocks.BlockSvg.prototype.jsonInit;
+
+  ScratchBlocks.BlockSvg.prototype.jsonInit = function (json) {
+    if (!addon.self.disabled && opcodeToSettings[this.type] && addon.settings.get(opcodeToSettings[this.type])) {
+      originalJsonInit.call(this, {
+        ...json,
+        outputShape: ScratchBlocks.OUTPUT_SHAPE_SQUARE,
+      });
+    } else {
+      originalJsonInit.call(this, json);
+    }
+  };
+
+  // taken from custom-block-shape
+  function updateAllBlocks() {
```

Comment:
This function is now its own JavaScript module (/addons/custom-block-shape/update-all-blocks.js). We should use it instead of duplicating the code.